lib/blockscore/actions/update.rb

```diff
@@ -27,20 +32,39 @@ module Update
 
       def_delegators 'self.class', :endpoint, :patch
 
+      # May be deprecated in future releases, prefer :update.
       def save!
         if persisted?
-          patch("#{endpoint}/#{id}", filter_params)
+          patch(update_url, filter_params)
           true
         else
           super
         end
       end
 
+      def update(params)
+        update!(params)
+        true
+      rescue
```

Comment:
Would you want it to just pass the exception on, or do you have something else in mind? 

My point of reference was: https://github.com/BlockScore/blockscore-ruby/blob/master/lib/blockscore/base.rb#L46